topics/statistics/tutorials/dome/tutorial.md

```diff
@@ -13,11 +13,530 @@ key_points:
+The goal is to find a balance between underfitting and overfitting, ensuring that the model captures the underlying patterns in the data while still generalizing well to unseen data. 
+Proper evaluation, regularization, and tuning of the model during the fitting process are crucial to achieving a good fit.
+
+
+**Key Questions**
+
+- Is *p* much larger than the number of training points and/or is *f* large (for example, in classification is *p >> (N<sub>pos</sub> + N<sub>neg</sub>)* and/or *f > 100*)? 
+- If yes, how was __overfitting__ ruled out? 
+- Conversely, if the number of training points is much larger than *p* and/or *f* is small (for example, *(N<sub>pos</sub> + N<sub>neg</sub>) >> p*  and/or *f < 5*), how was __underfitting__ ruled out?
+
+> **From Example Publication**  
+> Single input ML methods are used with default parameters.    Optimization is a simple majority.
+
+## 2.7 Regularization
+
+
+Regularization is a technique used to prevent overfitting by adding a penalty to the loss function, which discourages the model from becoming too complex. Common regularization techniques include:
+
+
+- L1 Regularization (Lasso): Adds a penalty proportional to the absolute value of the coefficients. It encourages sparsity, setting some coefficients to zero.
+- L2 Regularization (Ridge): Adds a penalty proportional to the square of the coefficients, discouraging large coefficients and thus reducing model complexity.
+- Dropout (in neural networks): Randomly drops a percentage of neurons during training, which helps prevent overfitting by forcing the network to generalize.
+
+
+**Key Questions**
+
+- Were any __overfitting prevention techniques__ used (for example, early stopping using a validation set)?
+
+- If yes, __which__ ones?
+
+> **From Example Publication**  
+> No. 
+
+
+## 2.8 Availability of configuration
+
+
+
+Availability of configuration refers to the accessibility and transparency of the settings, parameters, and options that can be adjusted or customized in a ML model or system. 
+These configurations control how the model is trained, how it makes predictions, and how it operates in different environments. 
+Ensuring that the configuration is available, flexible, and easy to modify is important for reproducibility, fine-tuning, and deployment of models.
+
+
+**Key Questions**
+
+- Are the hyperparameter configurations, optimization schedule, model files and optimization parameters __reported__? 
+- If yes, __where__ (for example, URL) and __how__ (license)?
+
+> **From Example Publication**  
+> Not applicable.
+
+   
+
+[//]: # (From Docs: Chapter 03)
+
+Good overall performance and the model's ability to generalize well to unseen data are crucial factors that significantly impact the applicability of any proposed ML research. However, several other important aspects related to ML models must also be considered.
+
+
+> **Note: Further Reading**  
```

Comment:
further reading could be in a comment box, or a details box (collapsed by default)

```
> <comment-title>Further Reading</comment-title>
> your text here..
{: .comment}
```

(replace comment with details if you want the box collapsed by default)